modules/ossm-excluding-namespaces-from-cluster-wide-mesh-cli.adoc

```diff
@@ -25,7 +25,12 @@ $ oc -n istio-system edit smcp <name> <1>
 ----
 <1> `<name>` represents the name of the `ServiceMeshControlPlane` resource.
 
-. Modify the YAML file so that the `spec.discoverySelectors` field of the `ServiceMeshMemberRoll` resource includes the discovery selector. The following example uses `istio-discovery: enabled`:
+. Modify the YAML file so that the `spec.discoverySelectors` field of the `ServiceMeshMemberRoll` resource includes the discovery selector. In the following example, Istiod discovers any namespace labeled `istio-discovery=enabled` or any namespace with name `bookinfo`, `httpbin` or `istio-system`.
++
+[NOTE]
+====
+When you decide which namespaces Istiod can discover, be careful not to include namespaces that may contain sensitive data or services.
```

Comment:
Suggested change: When configuring namespaces that Istiod can discover, be careful not to include namespaces that may contain sensitive services that should not be exposed to the rest of the mesh.

Reply:
done